script/build_docs.sh

```diff
@@ -17,17 +17,19 @@ else
     echo "$docs_dir already exists. Not cloning."
 fi
 
-
-index="${GOPATH%%:*}/src/${path}/index.asciidoc"
-
-echo "Building docs for ${name}..."
-echo "Index document: ${index}"
-
-dest_dir="$html_dir/${name}"
-mkdir -p "$dest_dir"
-params="--chunk=1"
-if [ "$PREVIEW" = "1" ]; then
-  params="--chunk=1 -open chunk=1 -open"
-fi
-$docs_dir/build_docs.pl $params --doc "$index" -out "$dest_dir"
-
+index_list="$(find ${GOPATH%%:*}/src/$path -name 'index.asciidoc')"
+for index in $index_list
+do
+  echo "Building docs for ${name}..."
+  echo "Index document: ${index}"
+  index_path=$(basename $(dirname $index))
+  echo "Index path: $index_path"
+
+  dest_dir="$html_dir/${name}/${index_path}"
+  mkdir -p "$dest_dir"
+  params="--chunk=1"
+  if [ "$PREVIEW" = "1" ]; then
+    params="--chunk=1 -open chunk=1 -open"
```

Comment:
Not sure what will happen here as `-open` actually opens a browser window.

Reply:
good point! 
I tried it out with two `index.asciidoc` files. The script builds the docs for the first one and opens it in the browser. Once this is interrupted the second one is built and opened in the browser. I thought this workflow is proper, but please let me know if you have concerns.

Reply:
Thanks for testing. SGTM.